v21.2/stream-data-out-of-cockroachdb-using-changefeeds.md

```diff
@@ -810,6 +810,119 @@ In this example, you'll set up a changefeed for a single-node cluster that is co
     $ cockroach quit --insecure
     ~~~
 
+### Create a changefeed connected to a webhook sink
+
+{{site.data.alerts.callout_info}}
+[`CREATE CHANGEFEED`](create-changefeed.html) is an [enterprise-only](enterprise-licensing.html) feature. For the core version, see [the `CHANGEFEED FOR` example above](#create-a-core-changefeed).
+{{site.data.alerts.end}}
+
+{% include {{ page.version.version }}/cdc/webhook-beta.md %}
+
+<span class="version-tag">New in v21.2:</span> In this example, you'll set up a changefeed for a single-node cluster that is connected to a local HTTP server via a webhook. For this example, you'll use an [example HTTP server](https://github.com/cockroachlabs/cdc-webhook-sink-test-server/tree/master/go-https-server) to test out the webhook sink.
+
+1. If you do not already have one, [request a trial enterprise license](enterprise-licensing.html).
+
+2. Use the [`cockroach start-single-node`](cockroach-start-single-node.html) command to start a single-node cluster:
```

Comment:
oh this might answer my question above.  might be a bit much to ask this doc to also handle the Cloud case in one go

Reply:
Yes, right now, I'm not sure how to move forward with mentions of Cloud in the bulk/IO/cdc docs that are under the current CRDB nav.... or if there is a plan for this